sdk/cosmos/azure-cosmos-spark_3_2-12/src/main/scala/com/azure/cosmos/spark/CosmosCatalogBase.scala

```diff
@@ -52,7 +52,7 @@ class CosmosCatalogBase
 
     /**
      * Called to initialize configuration.
-     * <p>
+     * <br/>
```

Comment:
are these intentional changes?

Reply:
yes - IntelliJ and CI checks for some files complaint about <p> or just empty lines.